tests/components/progettihwsw/test_config_flow.py

```diff
@@ -0,0 +1,43 @@
+"""Test the ProgettiHWSW Automation config flow."""
+from homeassistant import config_entries, setup
+from homeassistant.components.progettihwsw.config_flow import UnexistingBoard
+from homeassistant.components.progettihwsw.const import DOMAIN
+
+from tests.async_mock import patch
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] == {}
+
+    result2 = await hass.config_entries.flow.async_configure(
+        result["flow_id"],
+        {"host": "2.238.194.163:8085"},
```

Comment:
host shouldnt include port, need to ask for port if you want to support non-standard ports

Reply:
I added port field to config flow.